tests/specifications/environment_slurm.yaml

```diff
@@ -11,6 +11,6 @@ spark:
   workers:
     num_workers: 1  # num_workers + 1 nodes will be requested
     cpus_per_node: 1
-    mem_per_node: 1  # GB
-    time_limit: 1  # hours
+    mem_per_node: 1.5  # GB
```

Comment:
nit: maybe 0.1GB?

Reply:
we need the spark workers to have at least as much memory as the implementations to accept jobs and we need enough implementation memory to run the dummy pipelines. I'll try to constrain this a bit more though